src/Kunstmaan/MultiDomainBundle/Router/DomainBasedLocaleRouter.php

```diff
@@ -9,23 +9,12 @@
 use Symfony\Component\Routing\Generator\UrlGenerator;
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 use Symfony\Component\Routing\Matcher\UrlMatcher;
-use Symfony\Component\Routing\Route;
 use Symfony\Component\Routing\RouteCollection;
 
 class DomainBasedLocaleRouter extends SlugRouter
 {
-    /** @var RouteCollection */
-    protected $routeCollectionMultiLanguage;
```

Comment:
If this property is not needed anymore we need to deprecate it so it can be removed in 8.0